packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol

```diff
@@ -201,13 +157,20 @@ contract AssetMappings is IAssetMappings, Initializable{
             address currentAssetAddress = inputAsset.asset;
             validateAddAssetMapping(inputAsset);
 
-            inputAsset.baseLTV = uint256(inputAsset.baseLTV).convertToPercent().toUint64();
-            inputAsset.liquidationThreshold = uint256(inputAsset.liquidationThreshold).convertToPercent().toUint64();
-            inputAsset.liquidationBonus = uint256(inputAsset.liquidationBonus).convertToPercent().toUint64();
-            inputAsset.borrowFactor = uint256(inputAsset.borrowFactor).convertToPercent().toUint64();
+            (
+                inputAsset.baseLTV, 
+                inputAsset.liquidationThreshold,
+                inputAsset.liquidationBonus,
+                inputAsset.borrowFactor
+            ) = Helpers.convertCollateralParamsPercent(
```

Comment:
Should dont be converted to percent on-chain, should be already in percent format in params, and just validated.

Reply:
I see the validity in your concern. I don't expect that we will change the decimals from 18, so it's fair to do as you suggest